fw/apm.c

```diff
@@ -1953,6 +2107,13 @@ static TfwCfgSpec tfw_apm_specs[] = {
 		.allow_repeat	= true,
 		.cleanup	= tfw_cfgop_apm_cleanup_server_failover,
 	},
+	{
+		.name		= "health_stat_server",
```

Comment:
Is there a description for this in `/etc/tempesta_fw.conf` in this PR?

Reply:
No, like all other new directives, thanks! Fixed.